configure.ac

```diff
@@ -427,50 +440,44 @@ AS_CASE([$ocaml_cv_cc_vendor],
 # defaults to gnu89, which is not C99.  Clang defaults to gnu99 or
 # gnu11, which is fine.
 
-# Note: the vendor macro can not recognize MinGW because it calls the
-# C preprocessor directly so no compiler specific macro like __MING32__
-# is defined. We thus catch MinGW first by looking at host and examine
-# the vendor only as a fall-back. We could put tis part of the logic
-# in the macro itself, too
```

Comment:
This change is not strictly necessary, incidentally - it's just that with correct detection of the pre-processor, it is possible to identify mingw-w64 gcc.